### DISCUSSION TITLE
Cannot make `dynamic_linking` work following official docs

### DISCUSSION THREAD

**Discussion by SpencerMelo** [bevyengine/bevy#14181] [Q&A]
## Description
Hello, I'm trying to use bevy, however even with a "empty" project following the "Getting started" from Bevy site, the compile time is incredible high, and I see bevy getting compiled every single time.

I saw the optimizations available here: https://bevyengine.org/learn/quick-start/getting-started/setup/#compile-with-performance-optimizations.

However, even following it, when I run `cargo run` it do all the compilation of bevy again.
Does anyone faced the same or have any clue?

## Details

OS: Windows 10
cargo: 1.79.0 (ffa9cf99a 2024-06-03)
bevy: 0.14.0
rustc: 1.79.0 (129f3b996 2024-06-10)

### Cargo.toml
```
[package]
name = "space-farm"
version = "0.1.0"
edition = "2021"

[dependencies]
bevy = { version = "0.14.0", features = ["dynamic_linking"] }

[profile.dev]
opt-level = 1

[profile.dev.package."*"]
opt-level = 3
```

**Comment by SpencerMelo** [comment]
Note: I've done the same thing on Fedora and it works properly.

**Comment by trung1-dothanh** [comment]
Most of times I get this problem is because rust-analyze changed my build dir. Like I did `cargo run` with config A, then rust-analyzer run with config B, then when I build again everything need to be rebuilt with config A.
